external/iotivity/iotivity_1.2-rel/resource/c_common/iotivity_config.h

```diff
@@ -17,7 +17,7 @@
 
 #define HAVE_FCNTL_H 1
 
-#define HAVE_MEMORY_H 1
+//#define HAVE_MEMORY_H 1
```

Comment:
should it be commented or set to 0?

Reply:
It is used as below, we should comment  it.
```c
#ifdef HAVE_MEMORY_H
#include <memory.h>
#endif
```